src/v0/destinations/lambda/transform.js

```diff
@@ -70,7 +78,7 @@ function responseBuilderSimple(inputs, destConfig) {
   inputs.forEach(input => {
     const message = {
       payload: JSON.stringify(input.message),
-      destConfig
+      destConfig: getFieldsForDelivery(destConfig)
```

Comment:
Is this still left here for backward compatibility?

Reply:
yes otherwise we need to do coordinated deployment which is difficult to manage.